### ISSUE TITLE
The torrent detail view does not show the number of seeders/leechers in the `Trackers` section

### ISSUE THREAD

**Issue by josecelano** [torrust/torrust-index-gui#529]
It works on the right-top side (1 seeder) but it does not show that seeder at the bottom.

![image](https://github.com/torrust/torrust-index-gui/assets/58816/79435b75-20ae-4be5-8eda-f4a6a82d883a)

The data from the endpoint is OK:

```json
{
    "data": {
        "torrent_id": 5,
        "uploader": "creamenjoyer",
        "info_hash": "ec83837e5e828774d700dfa31636c40e89e079c3",
        "title": "The Floorwalker - Charlie Chaplin",
        "description": "1916 Classic\r\n\r\n![banner](https://upload.wikimedia.org/wikipedia/commons/5/50/The_Floorwalker_%28poster%29.jpg)",
        "category": {
            "id": 1,
            "category_id": 1,
            "name": "movies",
            "num_torrents": 2
        },
        "upload_date": "2024-04-01 15:04:56",
        "file_size": 84881060,
        "seeders": 1,
        "leechers": 0,
        "files": [
            {
                "path": [
                    "CC_1916_05_15_TheFloorwalker_512kb.mp4"
                ],
                "length": 84881060,
                "md5sum": null
            }
        ],
        "trackers": [
            "udp://tracker.torrust-demo.com:6969"
        ],
        "magnet_link": "magnet:?xt=urn:btih:ec83837e5e828774d700dfa31636c40e89e079c3&dn=The%20Floorwalker%20-%20Charlie%20Chaplin&tr=udp%3A%2F%2Ftracker.torrust-demo.com%3A6969",
        "tags": [
            {
                "tag_id": 1,
                "name": "tag1"
            }
        ],
        "name": "CC_1916_05_15_TheFloorwalker_512kb.mp4",
        "comment": null,
        "creation_date": 1711983706,
        "created_by": "qBittorrent v4.6.4",
        "encoding": null,
        "canonical_info_hash_group": [
            "ec83837e5e828774d700dfa31636c40e89e079c3"
        ]
    }
}
```

**Comment by hungfnt** [comment]
Should the Trackers Tab display the number of seeders/leechers for each individual tracker, or should it show the total number of seeders/leechers across all trackers?

**Comment by josecelano** [comment]
> Should the Trackers Tab display the number of seeders/leechers for each individual tracker, or should it show the total number of seeders/leechers across all trackers?

Hi @ngthhu, I did not implement that feature. I don't know what @mickvandijke's intention was. It makes sense to show seeders/leechers only for each tracker. However, we do not have that info for other trackers, at least now. We have discussed in the past having multiple associated trackers. In that case, it would be useful. I would suggest just removing that column for the time being. When I opened the issue I thought we could just put the same number, but now I think it does not make sense to show that column.
